mesh-delivery/ios/AVPlayerMeshDelivery/README.md

```diff
@@ -143,9 +143,9 @@ guard let deliveryUrl = deliveryClient.localManifestURL else {
 let playerItem = AVPlayerItem(asset: AVURLAsset(url: deliveryUrl))
 ```
 
-### 5. Play the stream
+### 5. Link the player with PlayerInteractor THEN play the stream
```

Comment:
Maybe don't capitalize the "THEN"? It might come up as rude, no ?

Reply:
Can we split them as two steps?

Reply:
removed "THEN" and split them into 2 steps in [70f0d91](https://github.com/streamroot/integration-samples/pull/53/commits/70f0d9140cdb0883b6a355094def510c2e0eb9fa)